### DISCUSSION TITLE
Relating Curb Zones to Their Curb Spaces via IDs

### DISCUSSION THREAD

**Discussion by jacobmalleau** [openmobilityfoundation/curb-data-specification#91] [Data Specification]
It seems like there is a way to relate Curb Zones to their Curb Areas and vice versa: Curb Zone has an optional attribute curb_area_id and Curb Area has a required attribute for curb_zone_ids.
You can also relate a Curb Space to its Curb Zone: Curb Space has a required curb_zone_id.
However there does not seem to be a way to reference what Curb Spaces are in a Curb Zone: i.e. an optional curb_space_ids attribute in Curb Zone.
My question is: is there a good reason we don't have this reference? 
I see value in this as it completes referencing between curb areas <-> curb zones <-> curb spaces.
It is also helpful if you want to quickly reference information on the curb spaces within a curb zone (although I see how you could do this by searching the curb_zone_id for all Curb Spaces.
I don't see harm in adding this in, especially if its an optional attribute.

Meant to bring this up today in the meeting, but didn't have time.

**Comment by schnuerle** [comment]
This could be a new optional field. 

So right now it's `areas <-> zones <- spaces` but it could be `areas <-> zones <-> spaces`.

I think the idea was to keep it referencial one way to avoid any data issues that could cause a conflict where you don't know what is true, eg IDs that don't match up and down stream. Which I guess could come up between zones and areas now.

**Reply by schnuerle** [reply]
Looking at it now, I think this was just an oversight.  Added a new optional field at the end of the table [here](https://github.com/openmobilityfoundation/curb-data-specification/blob/release-1.0.0/curbs/README.md#curb-zone).
